edge-modules/snitch/src/report.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use std::collections::HashMap;
+use std::io::Write;
+
+use bytes::Bytes;
+use chrono::{DateTime, Utc};
+use libflate::finish::AutoFinish;
+use libflate::gzip::Encoder as GzipEncoder;
+use tar::{Builder as TarBuilder, Header as TarHeader};
+
+use error::Result;
+use influx::QueryResults;
+
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Interval {
```

Comment:
Are we planning to add the missing message numbers?

Reply:
The analyzer does not return that at this time. Is that useful? Shouldn't just the start/end times be sufficient to help debug issues?